src/shared/cmpattern/cmpattern_teamdetector.cpp

```diff
@@ -506,6 +506,7 @@ void TeamDetector::findRobotsByModel(::google::protobuf::RepeatedPtrField< ::SSL
     robots->RemoveLast();
   }
 
+  delete markers;
```

Comment:
This should be `delete[]` . I'm a bit surprised that I didn't notice this given that clang is complaining now.